branchLayout/impl/build.gradle.kts

```diff
@@ -8,5 +8,5 @@ dependencies {
 junit()
 lombok()
 slf4jLambdaApi()
-slf4jSimpleTest()
+slf4jSimple("test")
```

Comment:
I see `test` here and `testFixtures` in the other place, and I assume there are some other options to... would it make sense to extract them to some enum?

Reply:
I'd leave it as is, even Gradle Kotlin DSL uses strings to denote these scopes... if there's any typo, we'll get a build error immediately